.github/workflows/go.yml

```diff
@@ -0,0 +1,37 @@
+name: Go
+on: [push, pull_request]
+jobs:
+
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Set up Go 1.14
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.14.4
+      id: go
+
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: build openebsctl
+      run: make openebsctl
+
+  lint:
+    name: Golint
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Set up Go 1.14
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.14.4
+      id: go_1
+
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: run golint
+      run: make golint
```

Comment:
We can add the verify deps as well ?

Reply:
yes, I think that would be good too, will just add it :)